src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
+    return null;
+  }
+
+  const formatted = {};
```

Comment:
When naming, it is better to give more informative names, for example `formattedRobot` or `invertedRobot`.